client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss

```diff
@@ -578,6 +578,15 @@ $design-button-primary-color: rgb(17, 122, 201);
 					}
 				}
 			}
+
+		}
+
+		.design-picker-design-title__theme-tier-badge {
+			gap: 8px;
+
+			&:empty {
+				display: none;
+			}
```

Comment:
What do you think about adding a comment explaining why we hide it? Although I think that the component returning an empty `div` is actually a bug and we should fix that instead.

Reply:
> I think that the component returning an empty div is actually a bug and we should fix that instead.

Seems reasonable, I can try to address that on a separate PR.

Reply:
Fix PR: https://github.com/Automattic/wp-calypso/pull/86659